datatypes/src/primitives/time_instance.rs

```diff
@@ -60,7 +54,13 @@ impl TimeInstance {
     }
 
     pub fn now() -> Self {
-        Self::from(chrono::offset::Utc::now())
+        Self::from(DateTime::now())
+    }
+
+    /// Converts a `TimeInstance` to a `DateTime`.
```

Comment:
shouldn't this return a Result instead of an Option? you convert it into a result in TimeStep anyway

Reply:
But isn't the Error for `TimeStep` more specific that this one? It essentially `None`s if it overflows.